sql/2020/08_Accessibility/common_video_attributes.sql

```diff
@@ -0,0 +1,51 @@
+#standardSQL
+# Video elements attribute usage
+CREATE TEMPORARY FUNCTION getUsedAttributes(payload STRING)
+RETURNS ARRAY<STRING> LANGUAGE js AS '''
+try {
+  const almanac = JSON.parse(payload);
+  return Object.keys(almanac.videos.attribute_usage_count);
+} catch (e) {
+  return [];
+}
+''';
+SELECT
+  client,
+  total_sites,
+
+  total_sites_with_video,
+  pct_sites_with_video,
+
+  attribute,
```

Comment:
Do you care about all attributes or only the ones related to a11y?

Reply:
Mostly those for a11y, but there are others which may be interesting to the authors like the `mute` attribute.

Since there's not that many results I decided it'd be easiest to give them all the data. Might spark their curiosity and give them ideas for even more interesting queries.